playbooks/roles/grafana/templates/ldap-certificate.crt.j2

```diff
@@ -0,0 +1 @@
+{{ grafana_auth_ldap_certificate }}
```

Comment:
instead of having an empty template use `copy: content: "{{ grafana_auth_ldap_certificate }}" ...`